deploy/crds/keda.sh_scaledjobs_crd.yaml

```diff
@@ -3,11 +3,26 @@ kind: CustomResourceDefinition
 metadata:
   name: scaledjobs.keda.sh
 spec:
+  additionalPrinterColumns:
```

Comment:
This seems to close #777 as well then or at least relate to it?

Reply:
I'll add more details in the following PR to fully cover #777